src/main/java/de/jutzig/github/release/plugin/UploadMojo.java

```diff
@@ -82,7 +82,7 @@ public class UploadMojo extends AbstractMojo implements Contextualizable{
 	/**
 	 * The release description
 	 */
-	@Parameter(property = "project.description")
+	@Parameter
```

Comment:
I'd suggest to drop `project.description` - from my experience, in 99% of the cases it is included in POM, but when looking at the release page on Github, we already have a context of what this codebase is about. So now that release notes will be generated, it might make less sense to have that repetitive part on top of that.

Should anyone still need it, there is always a way to say in the plugin configuration:

```
<description>${project.description}</description>
```